catalyst-gateway/bin/src/db/event/signed_docs/mod.rs

```diff
@@ -1,14 +1,32 @@
 //! Signed docs queries
 
+mod full_signed_doc;
+mod signed_doc_body;
 #[cfg(test)]
 mod tests;
 
-use super::EventDB;
+pub(crate) use full_signed_doc::FullSignedDoc;
+pub(crate) use signed_doc_body::SignedDocBody;
+
+use super::{common::query_limits::QueryLimits, EventDB, NotFoundError};
+use crate::jinja::{get_template, JinjaTemplateSource};
 
 /// Insert sql query
 const INSERT_SIGNED_DOCS: &str = include_str!("./sql/insert_signed_documents.sql");
 
-/// Make an insert query into the `event-db` by adding data into the `signed_docs` table
+/// Select sql query jinja template
+pub(crate) const SELECT_SIGNED_DOCS_TEMPLATE: JinjaTemplateSource = JinjaTemplateSource {
```

Comment:
Do these constants need to be `pub(crate)` ?  Will we use them outside the module?

Reply:
Yes, they are used inside `jinja` module when initialising all templates at once